plugins/livelog/livelog.go

```diff
@@ -20,19 +22,24 @@ type pluginProvider struct {
 
 type plugin struct {
 	plugins.PluginBase
-	log *logrus.Entry
+	log         *logrus.Entry
+	environment *runtime.Environment
```

Comment:
Just a note: I want to pass environment by value in the future. Not now, just a note.